draft-packages/collapsible/package.json

```diff
@@ -32,12 +32,13 @@
   "private": false,
   "license": "MIT",
   "dependencies": {
+    "@kaizen/component-base": "^1.0.0",
```

Comment:
Hmm...good question. I was more under the impression that it could later house more than just types so it's safer in here 🤔 But I suppose it makes more sense in the meantime to be a devDep. I'll move it :)